cmake/scripts/common/Macros.cmake

```diff
@@ -761,6 +762,9 @@ macro(core_find_versions)
   # unset variables not used anywhere else
   unset(version_list)
   unset(APP_APP_NAME)
+  unset(APP_COMPANY_NAME)
```

Comment:
https://github.com/xbmc/xbmc/blob/6c4ba80b694141d83968d66a15607c4e7d367de5/tools/windows/packaging/uwp/package.appxmanifest.in#L14